app/src/main/java/com/example/eric/socials/AddEvent.java

```diff
@@ -0,0 +1,133 @@
+package com.example.eric.socials;
+
+import android.app.Activity;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.net.Uri;
+import android.provider.MediaStore;
+import android.support.annotation.NonNull;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.Toast;
+
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.storage.FirebaseStorage;
+import com.google.firebase.storage.StorageReference;
+import com.google.firebase.storage.UploadTask;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class AddEvent extends AppCompatActivity implements View.OnClickListener{
+
+    EditText eventName;
+    EditText date;
+    EditText description;
+    Button createSocial;
+    ImageView eventImage;
+
+    private Intent data;
+
+    FirebaseDatabase database = FirebaseDatabase.getInstance();
+    DatabaseReference myRef = database.getReference("/socials");
+    private StorageReference mStorageRef;
+
+    private static FirebaseAuth mAuth;
+    private static FirebaseUser mUser;
+
+    @Override
+    public void onClick(View v){
+        switch (v.getId()) {
+            case R.id.createEventButton:
```

Comment:
Code in each case in this switch statement could be modularized into functions to make it easier to read. Like you could have createEventButtonClicked()